homeassistant/components/hive/services.yaml

```diff
@@ -1,24 +1,62 @@
 boost_heating:
+  name: Boost Heating
   description: Set the boost mode ON defining the period of time and the desired target temperature for the boost.
   fields:
     entity_id:
-      description: Enter the entity_id for the device required to set the boost mode.
-      example: "climate.heating"
+      name: Entity ID
+      description: Select entity_id to boost.
+      required: true
+      example: climate.heating
+      selector:
+        entity:
+          integration: hive
+          domain: climate
```

Comment:
Ideally, the integration should be adjusted to support `target` by adjusting these services to be entity services:

<https://developers.home-assistant.io/docs/dev_101_services#entity-services>

That would make these services more flexible, backward compatible with targets, allowing them to be used in Blueprints, area's and all these new goodness as well. Simplifies the code and removes the need for manually looking up/resolving entity IDs